Resources/Prototypes/Entities/Structures/Machines/chem_master.yml

```diff
@@ -85,7 +85,8 @@
           - PillCanister
   - type: SolutionContainerManager
     solutions:
-      buffer: {}
+      buffer:
+        maxVol: 1000
   - type: DumpableSolution
     solution: buffer
     unlimited: true
```

Comment:
```suggestion
    unlimited: false
```
Does this need to be changed to false?
![image](https://github.com/space-wizards/space-station-14/assets/121047731/83c444ed-0a71-48de-81da-e259a4492154)